apps/src/netsim/NetSimLogBrowser.jsx

```diff
@@ -0,0 +1,159 @@
+/** @file Modal dialog for browsing any logs in the simulation. */
+import React from 'react';
+import Dialog, {Title, Body} from '../templates/Dialog';
+import Packet from './Packet';
+import NetSimLogBrowserFilters from './NetSimLogBrowserFilters';
+import NetSimLogBrowserTable from './NetSimLogBrowserTable';
+
+// We want the table to scroll beyond this height
+const MAX_TABLE_HEIGHT = 500;
+
+const style = {
+  scrollArea: {
+    maxHeight: MAX_TABLE_HEIGHT,
+    overflowY: 'auto'
+  }
+};
+
+const NetSimLogBrowser = React.createClass({
+  propTypes: Object.assign({}, Dialog.propTypes, {
+    i18n: React.PropTypes.objectOf(React.PropTypes.func).isRequired,
+    canSetRouterLogMode: React.PropTypes.bool,
+    isAllRouterLogMode: React.PropTypes.bool,
+    setRouterLogMode: React.PropTypes.func.isRequired,
+    localAddress: React.PropTypes.string,
+    currentTrafficFilter: React.PropTypes.string.isRequired,
+    setTrafficFilter: React.PropTypes.func.isRequired,
+    headerFields: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    logRows: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    renderedRowLimit: React.PropTypes.number
+  }),
+
+  getDefaultProps() {
+    return {
+      isAllRouterLogMode: true,
+      currentTrafficFilter: 'none'
+    };
+  },
+
+  render() {
+    return (
+      <Dialog fullWidth {...this.props}>
+        <Title>Log Browser</Title>
```

Comment:
I totally forgot to do this - there's some actual functionality around the title reflecting the current filter settings.  I'll port it over.